changes/gpu-compute-model.tex

```diff
@@ -1,8 +1,93 @@
 \subsection[GPU Compute Model]{GPU Compute Model\footnote{by Anthony Gutierrez}}
 \label{sec:gpu}
 
-GPUs have become an important part of the system design for high-performance computing, machine learning, and many other workloads.
-Thus, we have integrated a compute-based GPU model into gem5~\cite{GutierrezBeckmann2018-amdAPU}.
+GPUs have become an important part of the system design for high-performance
+computing, machine learning, and many other workloads. Thus, we have integrated
+a compute-based GPU model into gem5~\cite{GutierrezBeckmann2018-amdAPU}. The
+GPU model is based on AMD's Graphics Core Next (GCN) architecture
+\footnote{http://amd.com/en/technologies/gcn}, and at the time of this writing
```

Comment:
```suggestion
- at the time of this writing
+ currently
```

(I can also submit a separate pull request for this if it requires you going through legal again)

Reply:
Sorry, the suggested edit did not show up properly.  It was just replacing "at the time of this writing" with "currently", not removing the footnote.

Reply:
It's fine, I understood. Done.